org/ystia/experimental/consul/topologies/ConsulOnBS/types.yaml

```diff
@@ -154,13 +156,15 @@ topology_template:
       type: tosca.nodes.BlockStorage
       properties:
         size: "1 GIB"
-        device: "/dev/vdb"
```

Comment:
Another difference between Alien and the spec... Device is a property of the BlockStorage.
Please revert...